app/assets/src/api/amr.js

```diff
@@ -6,3 +6,13 @@ export const getAMRCounts = sampleIds =>
       sampleIds,
     },
   });
+
+export const getCARDIndex = accession =>
+  get("amr_heatmap/fetch_card_index.json");
+
```

Comment:
It looks like the error responses for getCARDIndex and getAroEntry are not being handled, but you can have a simple message like "Error loading blah". Over time the source data URLs will go down and change

Reply:
Errors are now shown in the sidebar!